data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -420,8 +428,15 @@ def clean_combined_dataset(project_id=None, dataset_id=None):
     sandbox_dataset_id = sandbox.create_sandbox_dataset(project_id=project_id,
                                                         dataset_id=dataset_id)
 
+    validation_dataset_id = bq_utils.get_latest_validation_dataset_id(
+        project_id)
+
+    unioned_dataset_id = bq_utils.get_unioned_dataset_id()
```

Comment:
Not necessarily, we could totally get those directly for sure. I think retrieving those variables inside the cleaning rule seems to be more pythonic than to pass them down. I remember we had a similar discussion about whether we should pass `sandbox_dataset_id` down to the cleaning rules and we agreed to pass it down. But now that we have so many dataset ids that need to be passed, I agree it may be a better idea to access those variables directly. 

However, what's confusing me is that if we can access those variables directly, why pass any variable at all e.g. `project_id` and `dataset_id`?